ASP.NET Core/Web/AspNetCoreTemplate.Web/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace AspNetCoreTemplate.Web.Areas.Identity.Pages.Account
+{
+    using System.Security.Claims;
+    using System.Threading.Tasks;
+
+    using AspNetCoreTemplate.Data.Models;
+    using Microsoft.AspNetCore.Authorization;
```

Comment:
Blank line before this namespace group

Reply:
https://github.com/NikolayIT/ASP.NET-MVC-Template/pull/36/commits/fa48e032e9993476f0f85e1ca621d40559fadbd0